docs/tutorials/using_apis/connectivity/light-control.md

```diff
@@ -63,95 +99,69 @@ An example of the `mbed_app.json` file:
     }
 ```
 
-The following tables show the values to use in the `mbed_app.json` file for your devices in different networks.
-
-- For a 6LoWPAN-ND based network, use `nsapi.default-mesh-type: LOWPAN`.
-- For a Thread-based network, use `nsapi.default-mesh-type: THREAD`.
-
-### 6LoWPAN-ND
-
-**nsapi.default-mesh-type: LOWPAN**
-
-|Device role|`nanostack.configuration` value|`mbed-mesh-api.6lowpan-nd-device-type` value|
-|-----------|-------------------------|------------------------------------|
-|Mesh router (default) | lowpan_router | NET_6LOWPAN_ROUTER |
-|Nonrouting device | lowpan_host | NET_6LOWPAN_HOST |
-
-### Thread
-
-**nsapi.default-mesh-type: THREAD**
-
-|Device role|`nanostack.configuration` value|`mbed-mesh-api.thread-device-type` value|
-|-----------|-------------------------|------------------------------------|
-|Mesh router (default) | thread_router | MESH_DEVICE_TYPE_THREAD_ROUTER |
-|Nonrouting device | thread_end_device | MESH_DEVICE_TYPE_THREAD_SLEEPY_END_DEVICE |
-
 #### Thread commissioning
 
-By default, the Thread application uses the static network link configuration defined in the [mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
+By default, the Thread application uses the static network link configuration defined in the [Mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
 If you want to commission a Thread device, see [how to commission a Thread device in practice](../reference/thread-tech.html#thread-commissioning).
 
-The Thread stack learns the network settings from the commissioning process and saves them to RAM memory. Therefore, the learned network settings are lost when you restart the device next time. To prevent re-commissioning, you can save the Thread configuration settings to an SD card as follows (only in `K64F`):
+The Thread stack learns the network settings from the commissioning process and, by default, saves them to RAM memory. Therefore, the learned network settings are lost every time you restart the device. To prevent the need to recommission, save the Thread configuration settings to an SD card instead of RAM (only for the `K64F`):<!--why? can't it work with other boards that have an SD card?-->
```

Comment:
@SeppoTakalo @jarlamsa Can you answer this?

Reply:
@artokin could you answer this? I think currently the example application limits the SD card usage only for K64F.

Reply:
It is just like @jarlamsa explained, the mesh-minimal supports SD card for K64F only. 
The SD card can be used for all boards but we have tested and configured SD card only for K64F.

Reply:
Does that mean to use this tutorial, you have to have a K64F? Or that you have to use SD with a K64F but you can use RAM for others?

Reply:
You can use this application with many boards and all of them are saving data to RAM. 
With K64F you have option to save data to SD card but it is not mandatory.